alf/experience_replayers/replay_buffer.py

```diff
@@ -90,6 +99,23 @@ def __init__(self,
             allow_multiprocess (bool): whether multiprocessing is supported.
             keep_episodic_info (bool): index episode start and ending positions.
                 If None, its value will be set to True if ``num_earliest_frames_ignored``>0
+            record_episodic_return (bool): If True, computes and stores episodic return for
+                every step in the episode upon episode completion.  The field
+                ``discounted_return'' stores the information.  When episodes are
+                incomplete, all steps get the ``default_return''.
+                NOTE:
+                1) Reward transformations like ``RewardClipping.minmax=(-1, 1)'' set in
+                    ``TrainerConfig.data_transformer_ctor'' have to be set manually for
+                    the ReplayBuffer to be consistent:
+                    ``ReplayBuffer.reward_clip=(-1,1)''.
+                2) Discount ``gamma'' needs to be set consistent with ``TDLoss.gamma''.
+                3) Assumes ``keep_episodic_info'' to be True.
+            default_return (float): The default values of ``discounted_return''
+                when the episode has not ended.  It should be a very small value for
+                value target lower bounding.
```

Comment:
Comment that its value should not be bigger than the smallest possible discounted return.
Also note that it can be 0 if reward is always non-negative.